fitbenchmarking/parsing/fitbenchmark_parser.py

```diff
@@ -299,9 +302,8 @@ def _get_starting_values(self):
         :return: Starting values for the function
         :rtype: list of OrderedDict
         """
-        # Mantid functions can have reserved keywords so ignore these as they
-        # are not parameters.
-        ignore = ['name', 'BinWidth', 'ties', 'Formula', 'constraints']
+        # SasView functions can have reserved keywords so ignore these
+        ignore = ['name']
 
         name_template = '{1}' if len(self._parsed_func) == 1 else 'f{0}_{1}'
```

Comment:
If `_get_starting_values` is now only called for Sasview problems, is `name_template` needed as (from my understanding anyway) the length of `self._parsed_func` should only ever be greater than 1 for multifit problems?

Reply:
Good catch, looks like we can just use `name_template='{1}'` here?

Reply:
Looking at how `starting_values` is set (lines 309-312), I think we can remove `name_template` altogether by replacing lines 309-312 with the following:
```
starting_values = [
            OrderedDict([(name, val)
                         for name, val in self._parsed_func[0].items()
                         if name not in ignore])]
```